apps/web/src/components/forms/signup.tsx

```diff
@@ -61,10 +63,12 @@ export const SignUpForm = ({ className }: SignUpFormProps) => {
     try {
       await signup({ name, email, password, signature });
 
-      await signIn('credentials', {
-        email,
-        password,
-        callbackUrl: '/',
+      router.push('/signin');
```

Comment:
Replaced the `signIn` function with a redirect to the `/signin` page, since the newly-registered users can't sign in until they confirm their email address.